modules/ROOT/pages/sftp/sftp-documentation.adoc

```diff
@@ -68,384 +78,456 @@ An FileSystemProvider which provides instances of SftpFileSystem from instances
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS |  +++A TimeUnit which qualifies the #responseTimeoutUnit attribute. Defaults to SECONDS+++ |  +++SECONDS+++ | {nbsp}
-| Host a| String |  +++The FTP server host, such as www.mulesoft.com, localhost, or 192.168.0.1, etc+++ |  | *x*{nbsp}
-| Port a| Number |  +++The port number of the SFTP server to connect on+++ |  +++22+++ | {nbsp}
-| Username a| String |  +++Username for the FTP Server. Required if the server is authenticated.+++ |  | {nbsp}
-| Password a| String |  +++Password for the FTP Server. Required if the server is authenticated.+++ |  | {nbsp}
-| Passphrase a| String |  +++The passphrase (password) for the identityFile if required. Notice that this parameter is ignored if #identityFile is not provided+++ |  | {nbsp}
-| Identity File a| String |  +++An identityFile location for a PKI private key.+++ |  | {nbsp}
+** DAYS |  A TimeUnit which qualifies the *Response Timeout* attribute.  Defaults to SECONDS. |  SECONDS | 
+| Host a| String |  The FTP server host, such as `www.mulesoft.com`, localhost, or 192.168.0.1, etc. |  | x
+| Port a| Number |  The port number of the SFTP server to connect on. |  22 | 
+| Username a| String |  Username for the FTP Server. Required if the server is authenticated. |  | 
+| Password a| String |  Password for the FTP Server. Required if the server is authenticated. |  | 
+| Passphrase a| String |  The pass phrase (password) for the identityFile if required. Notice that this parameter is ignored if #identityFile is not provided. |  | 
+| Identity File a| String |  An identityFile location for a PKI private key. |  | 
 | PRNG Algorithm a| Enumeration, one of:
 
 ** AUTOSELECT
 ** NativePRNG
 ** SHA1PRNG
 ** NativePRNGBlocking
-** NativePRNGNonBlocking |  +++The Pseudo Random Generator Algorithm to use+++ |  +++AUTOSELECT+++ | {nbsp}
-| Reconnection a| <<Reconnection>> |  +++When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy+++ |  | {nbsp}
-| Pooling Profile a| <<PoolingProfile>> |  +++Characteristics of the connection pool+++ |  | {nbsp}
+** NativePRNGNonBlocking |  The Pseudo Random Generator Algorithm to use. |  AUTOSELECT | 
+| Reconnection a| <<Reconnection>> |  When an app is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
```

Comment:
Kim had suggestions on this: deployment fails if the deployment connectivity test to each connector fails.

Reply:
In this case, there is only one connector. If we lose the "reconnection strategy" concept, the reason deployment fails makes less sense.